templates/fluent-resource/en-US/code-of-conduct.ftl

```diff
@@ -0,0 +1,113 @@
+### Translation file for page: https://www.rust-lang.org/policies/code-of-conduct
+
+## templates/policies/code-of-conduct.hbs
+
+coc-page-heading = Code of conduct
+
+coc-conduct-heading = Conduct
+coc-conduct-li1 =
+        We are committed to providing a friendly, safe and welcoming environment
+        for all, regardless of level of experience, gender identity and
+        expression, sexual orientation, disability, personal appearance, body
+        size, race, ethnicity, age, religion, nationality, or other similar
+        characteristic.
+coc-conduct-li2 =
+        Please avoid using overtly sexual aliases or other nicknames that might
+        detract from a friendly, safe and welcoming environment for all.
+coc-conduct-li3 =
+        Please be kind and courteous. There’s no need to be mean or rude.
+coc-conduct-li4 =
+        Respect that people have differences of opinion and that every design or
+        implementation choice carries a trade-off and numerous costs. There is
+        seldom a right answer.
+coc-conduct-li5 =
+        Please keep unstructured critique to a minimum. If you have solid ideas
+        you want to experiment with, make a fork and see how it works.
+coc-conduct-li6 =
+        We will exclude you from interaction if you insult, demean or harass
+        anyone. That is not welcome behavior. We interpret the term “harassment”
+        as including the definition in the
+        <a href="http://citizencodeofconduct.org/">Citizen Code of Conduct</a>;
+        if you have any lack of clarity about what might be included in that
+        concept, please read their definition. In particular, we don’t tolerate
+        behavior that excludes people in socially marginalized groups.
+coc-conduct-li7 =
+        Private harassment is also unacceptable. No matter who you are, if you
+        feel you have been or are being harassed or made uncomfortable by a
+        community member, please contact one of the channel ops or any of the
+        <a href="{{baseurl}}/governance/teams/moderation">Rust moderation
+        team</a> immediately. Whether you’re a regular contributor or a
+        newcomer, we care about making this community a safe place for you and
+        we’ve got your back.
+coc-conduct-li8 =
+        Likewise any spamming, trolling, flaming, baiting or other
+        attention-stealing behavior is not welcome.
+coc-conduct-link = Email The Moderation Team
+
+coc-moderation-heading = Moderation
+coc-moderation-p1 =
+        These are the policies for upholding our community’s standards of
+        conduct. If you feel that a thread needs moderation, please contact the
+        <a href="{{baseurl}}/governance/teams/moderation">Rust moderation
```

Comment:
Handlebars templates won't work here, make this a `{ $fluent-variable }` and use `textparam` to pass it in.